fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -170,16 +282,11 @@ def _pmf_predict(self, X, *, sensitive_features):
         :rtype: numpy.ndarray
         """
         check_is_fitted(self)
-        _, _, sensitive_feature_vector = _validate_and_reformat_input(
-            X, y=None, sensitive_features=sensitive_features, expect_y=False,
-            enforce_binary_labels=True)
-        positive_probs = _vectorized_prediction(
-            self._post_processed_predictor_by_sensitive_feature, sensitive_feature_vector,
-            self.estimator_.predict(X))
-        return np.array([[1.0 - p, p] for p in positive_probs])
-
-    def _threshold_optimization_demographic_parity(self, sensitive_features, labels, scores,
-                                                   grid_size=1000, flip=True):
+        return self.interpolated_thresholder_._pmf_predict(
+            X, sensitive_features=sensitive_features)
+        
+    def _threshold_optimization_for_utility(
```

Comment:
a better name here maybe

Reply:
I love this comment, because the name makes perfect sense to me, but it shows that it's not quite as clear as I thought! 
It does perform threshold optimization for a provided utility, though, so I wonder how we could find a better name.

Reply:
I will be renaming `utility` to `simple_metrics` as opposed to `equalized_odds` (which is the one "not simple" metric that requires a separate treatment).